src/EmitJs.re

```diff
@@ -1343,5 +1343,10 @@ let emitTranslationAsString =
        ~config,
        ~requires=finalEnv.requires,
      )
-  |> Emitters.toString(~separator="\n\n");
-};
\ No newline at end of file
+  |> Emitters.toStrings
```

Comment:
This is a good intermediate step.
A nice next step would be to start emitting code in a similar way to how the rest is done.
So have a data structure to represent things of the form:
```
export const Outer = { Medium: { Inner: { y: Outer_Medium_Inner_y,  },  },  };
```
And separate printing functions.

So one does not have to go check the result of printing for whitespaces or so. But can operate on the data structure before printing.

Reply:
As a first step, perhaps even just a list of pairs ("Outer", " { Medium: { Inner: { y: Outer_Medium_Inner_y,  },  },  }").

Then see if the second part of the pair is itself best represented with a data structure. Whether the added flexibility is useful.

Reply:
yes it think it's wasteful to print and string and then check if empty, nice suggestion